xcm/xcm-executor/src/config.rs

```diff
@@ -58,4 +58,12 @@ pub trait Config {
 
 	/// What to do when a response of a query is found.
 	type ResponseHandler: OnResponse;
+
+	/// The general asset trap - handler for when assets are left in the Holding Register at the
+	/// end of execution.
+	type AssetTrap: DropAssets;
+
+	/// The general asset trap - handler for when assets are left in the Holding Register at the
+	/// end of execution.
+	type AssetClaims: ClaimAssets;
```

Comment:
The docs for `AssetClaims` is exactly the same as `AssetTrap`, it doesn't seem to be intended?